src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -11,7 +11,13 @@
     <WarningsNotAsErrors>612,618</WarningsNotAsErrors>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <RazorRuntimeCompilation>false</RazorRuntimeCompilation>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings Condition="'$(RazorRuntimeCompilation)' == 'false'">enable</ImplicitUsings>
   </PropertyGroup>
+  <!-- For Unit Tests-->
+  <ItemGroup>
+    <InternalsVisibleTo Include="OrchardCore.Tests" />
```

Comment:
Are there new tests in this PR that require it?

Reply:
I think functional tests, but I'm not sure if they treat `Program` as internal by default or not

Reply:
It's due to Removal of Startup.cs, this allows Program.cs to visible to unit tests - as per one of the options in  https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0#basic-tests-with-the-default-webapplicationfactory